src/main/java/org/assertj/core/configuration/ConfigurationProvider.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * Copyright 2012-2017 the original author or authors.
+ */
+package org.assertj.core.configuration;
+
+import org.assertj.core.presentation.Representation;
+import org.assertj.core.presentation.StandardRepresentation;
+
+/**
+ * Provider for all the configuration settings / parameters within AssertJ.
+ * <p>
+ * All the configuration possibilities are registered via an SPI.
+ *
+ * @author Filip Hrisafov
+ * @since 2.9.0 / 3.9.0
+ */
+public final class ConfigurationProvider {
+
+  public static final ConfigurationProvider CONFIGURATION_PROVIDER = new ConfigurationProvider();
+
+  private final Representation defaultRepresentation;
+
+  public ConfigurationProvider() {
+    defaultRepresentation = Services.get(Representation.class, new StandardRepresentation());
```

Comment:
It won't, we (assertj) won't register the `StandardRepresentation` as an SPI. 

In `Services#get(Class, Object) ` the second parameter is a fallback, in case no SPI has been registered. There are already multiple `Representation` implementations in assertj, and it is not a problem. 

However, if you have 2 dependencies registering a `Representation` then it'll fail. What is the desired behaviour here?